servers/rendering_server.cpp

```diff
@@ -2312,6 +2312,13 @@ RenderingServer::RenderingServer() {
 					"rendering/vulkan/rendering/back_end",
 					PROPERTY_HINT_ENUM, "ForwardClustered,ForwardMobile"));
 
+	GLOBAL_DEF_RST("rendering/vulkan/forward_mobile/color_buffer_format", 0);
```

Comment:
Yes, then again, this should not be necesary. Mobile should always use RGBA10, while clustered should always used RGBA16F.

Reply:
Removed the setting :)